src/hotspot/share/runtime/globals.hpp

```diff
@@ -2009,6 +2009,11 @@ const int ObjectAlignmentInBytes = 8;
                                                                             \
   product(bool, ProfileExceptionHandlers, true,                             \
           "Profile exception handlers")                                     \
+                                                                            \
+  product(bool, AlwaysRecordEvolDependencies, true, EXPERIMENTAL,           \
+                "Unconditionally record nmethod dependencies on class "     \
+                "rewriting/transformation independently of the JVMTI "      \
+                " can_{retransform/redefine}_classes capabilities.")        \
```

Comment:
Probably match the indenting of previous delcarations?

Reply:
Done